onnx_xla/translations/translate_reshape.cc

```diff
@@ -0,0 +1,83 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+// TODO: ENFORCE_EQ macro
+// Takes in data tensor and shape input
+// Error if shape input is dynamic
+// Parses ONNX shape input (can have 0's and one -1)
+// Produces new_sizes attribute to make XlaOp for Reshape
+onnxStatus translateReshape(const Node& n,
+                            XlaBuilder& builder,
+                            ValueOpMap& valueToOp,
+                            const ValueLiteralMap& valueToLiteral) {
+  // Check if shape input is static
+  auto shapeDataIt = valueToLiteral.find(n.inputs().at(1));
+  if (shapeDataIt == valueToLiteral.end()) {  // TODO: ENFORCE_EQ
+    std::cerr << "Reshape operatore having dynamic shape input is not supported"
+              << std::endl;
+    return ONNXIFI_STATUS_UNSUPPORTED_VERSION;
+  }
+  // Compute # of elements in input data tensor (check validity of shapes)
+  const std::vector<Dimension>& originalShape = n.inputs().at(0)->sizes();
```

Comment:
The sizes are not guaranteed to be set (only when onnx shape inference is able to infer it). (not saying we need to support such case, but need to raise error to avoid later failing mysteriously).